### DISCUSSION TITLE
Skip Dislikes

### DISCUSSION THREAD

**Discussion by ASTRON42** [pear-devs/pear-desktop#1490] [Ideas]
Hey, first of all, thanks for the great project. Despite having YouTube Music Premium, I prefer using this app on my PC due to its excellent plugins.

However, one of the most frustrating aspects of YouTube Music for me is its inability to skip disliked songs. It just doesn't make sense – if I've disliked a track, it's obvious I don't want to hear it again.

If it's feasible without requiring too much time and effort, could you consider adding a plugin that allows skipping disliked songs? It would greatly enhance the user experience. Thanks!

**Comment by ArjixWasTaken** [comment]
Are you asking for a plugin that either:
- removes disliked songs from the queue
- skips disliked songs when they are played

Both achieve what you want but behave differently.

If by any chance you *want* to listen to a disliked song, the latter option would not allow that. <br/>
On the other hand, removing disliked songs from the queue doesn't skip the current playing song,
so it would allow you to manually listen to a disliked song.

**Comment by inson1** [comment]
@ASTRON42 hey there is plugin forr that now, is that what did you want? If yes, may you close it?

**Reply by ASTRON42** [reply]
I'm so pleased with the new feature that I'm truly thankful for the project. Keep it up, guys! :D

PS: Yeah, pls close it for me (thx btw)

**Reply by VanGeko** [reply]
its no longer working, since last update,
when pressing dislike on a song, it skips the song, but disliked songs who are already in a list, still played.
